File: Phase-1/src/main/java/SearchFields.java
import java.util.ArrayList;

public class SearchFields {
    private ArrayList<String> simpleWords;
    private ArrayList<String> minusWords;
    private ArrayList<String> plusWords;

    public SearchFields() {
        this.simpleWords = new ArrayList<>();
        this.plusWords = new ArrayList<>();
        this.minusWords = new ArrayList<>();
    }

    public ArrayList<String> getSimpleWords() {
        return simpleWords;
    }

    public ArrayList<String> getMinusWords() {
        return minusWords;
    }

    public ArrayList<String> getPlusWords() {
        return plusWords;
    }

    public void addSimpleWord(String word){
        this.simpleWords.add(word);
    }

    public void addPlusWord(String word){
        this.plusWords.add(word);
    }

    public void addMinusWord(String word){
        this.minusWords.add(word);
    }
}

File: Phase-5/SearchEngine/SearchFields.cs
using System.Collections.Generic;
using SearchEngine.Interfaces;

namespace SearchEngine
{
    public class SearchFields : ISearchFields
    {
        private List<string> _simpleWords;
        private List<string> _minusWords;
        private List<string> _plusWords;

        public SearchFields()
        {
            this._simpleWords = new List<string>();
            this._minusWords = new List<string>();
            this._plusWords = new List<string>();
        }

        public List<string> GetSimpleWords()
        {
            return _simpleWords;
        }

        public List<string> GetMinusWords()
        {
            return _minusWords;
        }

        public List<string> GetPlusWords()
        {
            return _plusWords;
        }

        public void AddSimpleWord(string word)
        {
            _simpleWords.Add(word);
        }

        public void AddPlusWord(string word)
        {
            _plusWords.Add(word);
        }

        public void AddMinusWord(string word)
        {
            _minusWords.Add(word);
        }
    }
}